docs/_documentations/workingwithtemplates.md

```diff
@@ -14,23 +14,26 @@ parent: usingcodewind
 
-When you create a new project, you can choose from the default set of templates available in Codewind or you can choose a template source of your own. You can add more development languages and sample projects by adding new templates. 
+When you create a new project, you can choose from the default set of templates available in Codewind, or you can choose a template source of your own. You can add more development languages and sample projects by adding new templates. 
 
 ## Managing templates
 
-Template management in Codewind is provided by the **Template Source Manager**. To open the **Template Source Manager**, right-click **Local**, and then click **Manage Template Sources**. The **Template Source Manager** appears. 
+Template management in Codewind is provided by the **Template Source Manager**. To open the **Template Source Manager**, right-click **Local** > click **Manage Template Sources**. The **Template Source Manager** appears. 
 
-Codewind templates are stored in the [codewind-resources/codewind-templates](https://github.com/codewind-resources/codewind-templates) GitHub repository. Standard Codewind Templates, the default set of templates from Kabanero Collections, and Appsody Stacks to develop applications with sharable technology stacks, are already included in Codewind and you can refer to those as examples.
+Codewind templates are stored in the [codewind-resources/codewind-templates](https://github.com/codewind-resources/codewind-templates)GitHub repository. Three examples are included in Codewind for your reference: 
+1. Standard Codewind Templates.
+2. Default templates from Kabanero Collections.
+3. Appsody Stacks to develop applications with sharable technology stacks. 
 
 Use the **Template Source Manager** to perform the following actions:
 1. To add a new template source to the table, click **Add new**. For more information, see [Adding your own template sources to Codewind](#adding-your-own-template-sources-to-codewind).
 2. To remove non-default template sources, click the trash icon. 
-3. To enable a template source to have its templates appear in the **Create Project** wizard so that projects of that style can be added to Codewind, toggle the `Enabled` slide to on. For example, enable at least one Appsody-style source before trying to add an Appsody project.
-4. To disable a set of templates so they do not appear in the **Create Project** wizard, toggle the `Enabled` slide to off.
+3. To enable a template source to have its templates appear in the **Create Project** wizard so those style projects can be added to Codewind, toggle the `Enabled` slide to `On`. For example, enable at least one Appsody-style source before adding an Appsody project.
+4. To disable a set of templates so they do not appear in the **Create Project** wizard, toggle the `Enabled` slide to `Off`.
```

Comment:
Not a big deal, but I think of **Enabled** and **Off** more like buttons, so I'd put them in bold instead of code phrase format.